lib/realtime/helpers.ex

```diff
@@ -53,7 +64,14 @@ defmodule Realtime.Helpers do
         application_name: "supabase_realtime"
       ],
       socket_options: socket_opts
-    )
+    ]
+
+    if ssl_enforced == true do
+      db_config ++ [ssl: true, ssl_opts: [verify: :verify_none]]
+    else
+      db_config
+    end
```

Comment:
This part of the code is used a few times, it might be helpful if we created a helper function for it

```elixir
  @spec enforce_ssl_opts(Keyword.t(), boolean()) :: Keyword.t()
  def enforce_ssl_opts(config, true) do
    config ++ [ssl: true, ssl_opts: [verify: :verify_none]]
  end

  def enforce_ssl_opts(config, _) do
    config
  end
```